docs/overview/index.md

```diff
@@ -1,17 +1,105 @@
 ---
 title: Tezos overview
 last_update:
-  date: 11 September 2023
+  date: 15 August 2024
 ---
 
 Tezos is an open-source, decentralized blockchain [created in 2014](https://tezos.com/whitepaper.pdf) by Arthur and Kathleen Breitman. It raised awareness and support in 2017 with its crowdfunding and launched the following year. Since its inception, Tezos has gone through multiple iterations of upgrades and development, staying true to its ethos -- "a blockchain designed to evolve".
 
 Tezos has robust applications ranging from NFTs, DeFi, and gaming to enterprise and government use cases.
 
+## What is Tezos?
```

Comment:
I'd suggest either removing this header or putting it at the beginning of the page, since the previous paragraph also talk about what is Tezos.

Reply:
Removed.